packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -118,12 +126,30 @@ export class LockedGoldWrapper extends BaseWrapper<LockedGold> {
   getValidatorFromAccount: (account: string) => Promise<Address> = proxyCall(
     this.contract.methods.getValidatorFromAccount
   )
+
+  /**
+   * Returns the account address given the signer for validating
+   * @param signer Address that is authorized to sign the tx as validator
+   * @return The Account address
+   */
+  validatorSignerToAccount: (signer: Address) => Promise<Address> = proxyCall(
```

Comment:
Is this consistent with what @nambrot is renaming things? If not, can we aim for consistency?

Reply:
i wanted to be on top of nam pr but couldn't...
so when we have that, we'll have to move some methods and have a consistent naming